src/core/router/index.js

```diff
@@ -26,23 +26,27 @@ const router = new Router({
 });
 router.beforeResolve((to, from, next) => {
   // Check if user is coming from a path that needs auth
-  if (
-    !from.meta.hasOwnProperty('requiresAuth') &&
-    to.meta.hasOwnProperty('requiresAuth')
-  ) {
+  if (store.state.wallet.address && to.meta.hasOwnProperty('requiresAuth')) {
```

Comment:
lets add `requireAuth: true` to routes that requires Auth, this workaround is not pretty

Reply:
actually switched it to `noAuth`